### DISCUSSION TITLE
Lots of Read Only Collabarators for Private Repo

### DISCUSSION THREAD

**Discussion by cadday** [community/community#22607] [New to GitHub]
<p>Hello,</p>
<p>We are selling our work on a marketplace and want to use Github for versioning and delivering updates. </p>
<p>Which plan should we buy for unknown number of read only collabarators for private repositories? </p>
<p>I have checked these posts and could not find what i was looking for.</p>
<p><a href="https://github.community/t/can-i-give-read-only-access-to-a-private-repo-from-a-developer-account/441">https://github.community/t5/How-to-use-Git-and-GitHub/Can-I-give-read-only-access-to-a-private-repo-from-a-developer/td-p/1967</a></p>
<p><a href="https://github.community/t/make-a-fork-of-a-private-repo-public/1400">https://github.community/t5/How-to-use-Git-and-GitHub/Make-a-fork-of-a-private-repo-public/m-p/13062/highlight/true#M4223</a></p>
<p><a href="https://github.community/t/private-repo-and-a-free-account/1005">https://github.community/t5/How-to-use-Git-and-GitHub/Private-Repo-and-a-Free-Account/m-p/7817#M2668</a></p>
<p>Regards.</p>

**Comment by MChevy422** [comment]
<p>Hey cadday ,</p>
<p>This post was moved to a different board that fits your topic of discussion a bit better. This means you’ll get better engagement on your post, and it keeps our Community organized so users are able to find information more easier.</p>
<p>As you’ll notice, your topic is now in the ‘How to use GIt and Github’ board. No action is needed on your part; you can continue the conversation as normal here on this board.</p>
<p>Let me know if you have any other questions or if I can help with anything else.</p>
<p>Greatly appreciated,</p>

**Comment by AndreaGriffiths11** [comment]
<p>Hi <a class="mention" href="/cadday">@cadday</a>,</p>
<p>Thank you for being here and for your interest in GitHub. If you haven’t already, please contact us at <a href="https://enterprise.github.com/contact" rel="nofollow noopener">https://enterprise.github.com/contact</a> so we can look into that further, and provide you with accurate pricing info.</p>
<p>Best,</p>
<p>Andrea</p>

**Comment by cadday** [comment]
<p>Hi Andrea,</p>
<p>Thanks for your answer. I have used another form 2-3 days ago and got no reply so far, so trying the forum. <a href="https://github.com/contact" rel="nofollow noopener">https://github.com/contact</a></p>
<p>Does Business Cloud provide this? We have tried with Team option but every collabarator takes a seat and that is economically unsustainable for us. </p>
<p>People are providing this service very widely and for a profit of ~$15 from a user. Paying $9 or more for month to Github to get a seat does not seem logical.</p>
<p>Here are two examples: </p>
<p><a href="http://withinpixels.com/themes/fuse-github" rel="nofollow noopener">http://withinpixels.com/themes/fuse-github</a></p>
<p><a href="https://keenthemes.com/metronic/?page=github" rel="nofollow noopener">https://keenthemes.com/metronic/?page=github</a></p>
<p>We hope that you can direct us to a solution.</p>
<p>Regards.</p>

**Comment by cadday** [comment, marked as answer]
<p>Hi,</p>
<p>People from Github help contacted and explained the situation.</p>
<p>Since Github <strong>changed it’s payment system from per-repository to per-user</strong> , it is not possible to have unknown number of collabs on a private repository.</p>
<p>I was also informed that they don’t pressure per-repository users to switch their system(which is good), i wonder why they don’t offer this to new customers. It would be a good thing to be able to buy what is already available. </p>
<p>If you have a similar need, best thing to do is have a developer account and create a private repo. You may invite unpaid users to your repo without a limitation. You will not be able to define permissions on user level though.</p>
<p>Regards.</p>
